backend/controllers/productController.js

```
@@ -1,7 +1,12 @@
 import asyncHandler from "express-async-handler";
 import productModel from "../models/productModel.js";
 import mongoose from "mongoose";
+import csv from "csvtojson";
+import XLSX from "xlsx";
+import fs from "fs";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)